src/index.js

```diff
@@ -1,21 +1,40 @@
-import _ from 'lodash';
-import printMe from './print.js';
+// import _ from 'lodash';
+// import printMe from './print.js';
+import Library from './modules/libraryModule.js';
 import './style.scss';
 
-function component() {
-  const element = document.createElement('div');
-  const btn = document.createElement('button');
+// const libraryBtn = document.getElementById('LibraryBtn');
+// const newBtn = document.getElementById('NewBtn');
+// const libraryPage = document.getElementById('Library');
+// const newPage = document.getElementById('New');
+// const libraryStr = 'library';
+// const hidden = 'hidden';
+// const newStr = 'new';
+// const contactStr = 'contact';
+// const pageSetup = () => {
+//   // newPage.className = hidden;
+//   // contactPage.className = hidden;
+// };
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.

Reply:
> * [ ]  Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.

Done, thank you :-)